src/wasm.h

```diff
@@ -1415,6 +1415,7 @@ struct BinaryLocations {
   // this as a simple struct with two elements (as two extra elements is the
```

Comment:
does this big comment need to be updated?

Reply:
Thanks, yes, done.